src/main/java/seedu/address/model/taskmanager/Deadline.java

```diff
@@ -47,5 +46,13 @@ public String toString() {
         return "Deadline has been added:\n"
                 + "[D][" + getStatusIcon() + "] " + this.description + "(by: " + this.dueBy + ")";
     }
+
+    public LocalDateTime getDueBy() {
+        return this.dueBy;
+    }
+
+    public String getDescription() {
```

Comment:
Can maybe abstract out the getDescription method to the task class to prevent repeated code